Atos20/Travel-tracker
# Feature/retrieve data using fetch

### PR Template:

#### What’s this PR do?  
- Adds the methods needed to request  data to the server by using the `fetch` in the `fetchHandle r class`
- Creates an `onStart` function in `index.js` that makes use of `Promise.all` to handle the `promises` returned by the server

#### Where should the reviewer start?  
 `index.js` and `fetchHandler.js`
#### How should this be manually tested?  
 run `npm start` and in the browser open `console`. The traveler class should be visible with all its properties.
#### What are the relevant tickets?  
 #4 
#### Screenshots (if appropriate) 
 ![Screen Shot 2020-09-19 at 12 01 49 AM](https://user-images.githubusercontent.com/56229864/93660364-54a67200-fa0b-11ea-9403-294ec26b6706.png)
